.github/workflows/e2e.ci.yaml

```diff
@@ -43,28 +43,32 @@ jobs:
         images:
           - name: recent
             oap-repo: ghcr.io/apache/skywalking/oap
-            oap-tag: cef4f6dcac5b14cd67f51a64ad08052bbc046653
+            oap-tag: 14fbd975be077082ed6fa604e05479dd9af0d995
             ui-repo: ghcr.io/apache/skywalking/ui
-            ui-tag: cef4f6dcac5b14cd67f51a64ad08052bbc046653
+            ui-tag: 14fbd975be077082ed6fa604e05479dd9af0d995
     name: Istio+Envoy Access Log Service(${{ matrix.images.name }})
     env:
       OAP_TAG: ${{ matrix.images.oap-tag }}
       OAP_REPO: ${{ matrix.images.oap-repo }}
       UI_TAG: ${{ matrix.images.ui-tag }}
       UI_REPO: ${{ matrix.images.ui-repo }}
-      SATELLITE_TAG: v64f1938e118e95b8d8f50756f666e4777ce6aca2
+      SATELLITE_TAG: v35bfaff6352b4dc351a706772796a1f79b651c14
       SATELLITE_REPO: ghcr.io/apache/skywalking-satellite/skywalking-satellite
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Setup go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: '1.16'
-      - name: Run Skywalking E2E Test
+          go-version: '1.20'
+      - name: Run Skywalking E2E Test (Elasticsearch as database)
         uses: apache/skywalking-infra-e2e@45584853d6f660102c523b1e9cb5815d12ae55d3
         with:
-          e2e-file: $GITHUB_WORKSPACE/test/e2e/e2e.yaml
+          e2e-file: $GITHUB_WORKSPACE/test/e2e/e2e-elasticsearch.yaml
+      - name: Run Skywalking E2E Test (BanyanDB as database)
+        uses: apache/skywalking-infra-e2e@45584853d6f660102c523b1e9cb5815d12ae55d3
+        with:
+          e2e-file: $GITHUB_WORKSPACE/test/e2e/e2e-banyandb.yaml
       - name: Run SWCK E2E Test
         uses: apache/skywalking-infra-e2e@afdf1cca0519d65bc480d8680b7a27f9b41fc421
         with:
```

Comment:
We should run e2e in the parallel mode. You could ref the main repo skywalking.yml to use `matrix` to define `name` and `config`.